src/components/tasks/list/taskCategoryComponent.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import CategoryHeader from '../../styled-components/common/CategoryHeader';
+import { TaskCategory } from '../../../models/task';
+import TaskCard from '../single/taskCard';
+import TaskCategoryCardContainer from '../../styled-components/tasks/TaskCategoryCardContainer';
+
+interface TaskCategoryProps {
+    category: TaskCategory;
+}
+
+const TaskCategoryComponent = ({ category }: TaskCategoryProps) => {
+    return (
+        <>
+            <CategoryHeader>{category.name}</CategoryHeader>
+            <TaskCategoryCardContainer>
+                {category.tasks &&
```

Comment:
`.tasks` cannot be undefined since you provide default value (empty list) for TaskCategory (inside splitTaskList func)